moveit2_rolling-ci-testing.repos

```diff
@@ -0,0 +1,17 @@
+repositories:
+  geometric_shapes:
+    type: git
+    url: https://github.com/ros-planning/geometric_shapes
+    version: ros2
+  moveit_msgs:
+    type: git
+    url: https://github.com/ros-planning/moveit_msgs
+    version: ros2
```

Comment:
Would it make sense to remove these from this file because technically the released versions are exactly the same (there are no changes).  The same applies to the other repos file.

Reply:
Not really, the point is that these can be ahead of the released version like moveit_resources and keeping them in here allows us to catch syncing issues early. The testing job should really run anything that is not released yet. The only alternative for keeping these in here would probably be downstream testing in the other repos. But still, we would have to add them again for the prerelease test, so why not just keep them in here?
Maybe we can remove the entries for the release build, because those should really match the version of the debians.

Reply:
I understand having them here when there are changes that are not yet released.  My point was that specifically in the case of these two there is no difference between the ros2 branch and the release.  Your point is that by leaving them here it would catch if there is a change.  As we control those repos though we will know when there is a change.

Reply:
I don't trust us enough to change this in time, I guess ;)